container/cypress/e2e/test-app/wc/wc-container.cy.js

```diff
@@ -131,5 +131,20 @@ describe('Web Container Test', () => {
             });
         });
     });
+    it('sendCustomMessage', () => {
+      cy.get(containerSelector)
+        .shadow()
+        .find('#customMessageDiv')
+        .should('have.text', 'Received Custom Message: ');
+
+      cy.get('#sendCustomMessageBtn')
+        .click()
+        .then(() => {
+          cy.get(containerSelector)
+            .shadow()
+            .find('#customMessageDiv')
+            .should('have.text', 'Received Custom Message: cool custom Message');
```

Comment:
why are there 2 **should('have.text)** tests for the same thing here . 1 is enough. I think scenario is. 
1. Click button to send message from  outside, and 
2. check if message received inside container mf